native/build.gradle

```diff
@@ -30,7 +30,7 @@ dependencies {
     implementation group: 'org.ballerinalang', name: 'ballerina-runtime', version: "${ballerinaLangVersion}"
     implementation 'com.google.guava:guava:32.1.1-jre'
     implementation 'org.apache.wss4j:wss4j-ws-security-dom:3.0.1'
-    implementation 'org.apache.santuario:xmlsec:3.0.2'
+    implementation 'org.apache.santuario:xmlsec:3.0.3'
```

Comment:
better to not hardcode the versions. check how this is done in other modules - https://github.com/ballerina-platform/module-ballerina-grpc/blob/master/native/build.gradle#L39

Reply:
Fixed in [8b1d56d](https://github.com/ballerina-platform/module-ballerina-soap/pull/113/commits/8b1d56d868b6eddcae4db67524ead4f4ca321689)